android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManagerImpl.kt

```diff
@@ -49,6 +51,7 @@ class RNCWebViewManagerImpl {
     private val HTML_ENCODING = "UTF-8"
     private val HTML_MIME_TYPE = "text/html"
     private val HTTP_METHOD_POST = "POST"
+    private val MAX_FILE_PATH_SIZE = 255
```

Comment:
How was this number determined? I have not found any information about there being such a small limit on Android.

Reply:
@TheAlmightyBob it's the `_POSIX_PATH_MAX` value. Actually, `_POSIX_PATH_MAX` is 256, but we need to consider the null ending byte in the char byte array, so we have 255 chars left for path name.